### DISCUSSION TITLE
Support Structed  Output?

### DISCUSSION THREAD

**Discussion by hlzhangxt** [sashirestela/simple-openai#251] [Q&A] [answered]
https://platform.openai.com/docs/guides/structured-outputs

**Comment by sashirestela** [comment, marked as answer]
Hi @hlzhangxt, thanks for your interest in simple-openai. That functionality already exists. Here you have a demo code:

https://github.com/sashirestela/simple-openai/blob/main/src/demo/java/io/github/sashirestela/openai/demo/ChatDemo.java#L150-L187

Just to let you know, it was delivered as part of the release 3.7.0 on August 19th
